frontend/src/metabase/lib/urls/utils.ts

```diff
@@ -14,6 +14,17 @@ function flattenParam([key, value]: [string, unknown]) {
   return [[key, value]];
 }
 
-export function extractQueryParams(query: Record<string, unknown>) {
```

Comment:
This function is only used in 2 places, so it's pretty safe to replace the call sites with the new function